prover/src/trace/trace_lde/default/mod.rs

```diff
@@ -279,30 +289,30 @@ where
     // build trace commitment
     let tree_depth = trace_lde.num_rows().ilog2() as usize;
     let trace_tree = info_span!("compute_execution_trace_commitment", tree_depth)
-        .in_scope(|| trace_lde.commit_to_rows());
-    assert_eq!(trace_tree.depth(), tree_depth);
```

Comment:
This assert was removed because we cannot get domain size from the vector commitment, right? If so, maybe we should add `VectorCommitment::domain_size(&self) -> usize` function/

Reply:
added